unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/Camera/Tests/InputSpikeFixerShould.cs

```diff
@@ -0,0 +1,61 @@
+using DCL.Camera;
+using NUnit.Framework;
+using UnityEngine;
+
+namespace CameraController_Test
+{
+    public class InputSpikeFixerShould
+    {
+        private CursorLockMode previousLockState;
+
+        [SetUp]
+        public void Setup()
+        {
+            previousLockState = Cursor.lockState;
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            Cursor.lockState = previousLockState;
+        }
+        
+        [TestCase(0, 0)]
+        [TestCase(10, 10)]
+        [TestCase(-10, -10)]
+        [TestCase(0.5f, 0.5f)]
+        public void WhenLockStateIsNotChangedValueWontChange(float inputValue, float expectedInputValue)
```

Comment:
"Input spike fixer should when lock state is not changed value wont change" is not a valid english sentence.

I suggest:
"NotChangeValueWhenSameLockStateIsSetTwice"

(or any name that matches the should sentence).